src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -525,8 +531,56 @@ export const OscapList = () => {
         <TextListItem component={TextListItemVariants.dd}>
           {oscapProfile}
         </TextListItem>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Kernel arguments:
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          <CodeBlock>
+            <CodeBlockCode id="code-content">
```

Comment:
Are the ids needed / working like a prop or something? What I mean is if they have a purpose as a variant setter or something similar.

If not we should either drop them or differentiate them so we can use the id for example as a unique identifier in the tests.